scripts/devops_tasks/tox_harness.py

```diff
@@ -35,27 +35,6 @@
 dependency_tools_path = os.path.join(root_dir, "eng", "dependency_tools.txt")
 
 
-def combine_coverage_files(targeted_packages):
```

Comment:
This code went entirely uncalled. We collect it directly from the package directories. The `.coverage` file gets dropped at package root. That doesn't get cleaned up like the rest of the `.tox` folder after all the environments for the package run.